e2e/checkin-flow.spec.ts

```
@@ -0,0 +1,114 @@
+    await page.goto('/queue/join');
+    
+    // Deve redirecionar para a home
+    await expect(page).toHaveURL('/');
+  });
+
+  test('deve exibir página de sucesso após entrar na fila', async ({ page }) => {
+    // Acessa a página de sucesso diretamente (simulando após join)
+    await page.goto('/queue/join/success?name=Doador%20Teste');
+    
+    // Verifica se a mensagem de sucesso está presente
+    const successContent = page.locator('text=Doador Teste, text=sucesso, .success-message, h1').first();
```

ESLint

[error] Replace `.locator('text=Doador·Teste,·text=sucesso,·.success-message,·h1')` with `⏎······.locator("text=Doador·Teste,·text=sucesso,·.success-message,·h1")⏎······`

(prettier/prettier)